### ISSUE TITLE
Can `info.json` show `space-age` as optional?

### ISSUE THREAD

**Issue by urda** [urda/quick-start-delight#8]
- See also: #6 

# Possible code change

```json
    "dependencies": [
        "base >= 2.0.0",
        "? space-age >= 2.0.0",
        "? elevated-rails >= 2.0.0", 
        "? quality >= 2.0.0"
    ],
```

# A server's `mod-list.json` sample

```json
{
  "mods":
  [

    {
      "name": "base",
      "enabled": true
    },

    {
      "name": "elevated-rails",
      "enabled": true
    },

    {
      "name": "quality",
      "enabled": true
    },

    {
      "name": "space-age",
      "enabled": true
    },

    {
      "name": "quick-start-delight",
      "enabled": true
    }
  ]
}
```

**Comment by urda** [comment]
Yeah with something like

```diff
diff --git a/changelog.txt b/changelog.txt
index d085bd1..ed272ff 100644
--- a/changelog.txt
+++ b/changelog.txt
@@ -1,6 +1,8 @@
 ---------------------------------------------------------------------------------------------------
 Version: 2.1.3
 Date: 20??.??.?? ??:00:00+00:00
+  Other:
+    - Mark `space-age` related mods as "optional"
 
 ---------------------------------------------------------------------------------------------------
 Version: 2.1.2
diff --git a/info.json b/info.json
index 13dd7ca..b48897f 100644
--- a/info.json
+++ b/info.json
@@ -7,7 +7,10 @@
     "contact": "foss@urda.engineering",
     "factorio_version": "2.0",
     "dependencies": [
-        "base >= 2.0.0"
+        "base >= 2.0.0",
+        "? space-age >= 2.0.0",
+        "? elevated-rails >= 2.0.0",
+        "? quality >= 2.0.0"
     ],
     "description": "Hit the ground running with a spidertron, robots, and more in your next game! This mod is meant for players who have already beaten the game at least once."
 }
```

We can get something like:

<img width="472" alt="image" src="https://github.com/user-attachments/assets/bebfccfd-7382-4724-8d63-d64f3a15dbe4">

and in Space Age:

<img width="417" alt="image" src="https://github.com/user-attachments/assets/bf9f8ae7-0980-4fe3-a88a-b71941c4010d">

**Comment by urda** [comment]
I think one could even

```json
    "dependencies": [
        "base >= 2.0.8",
        "? space-age >= 2.0.8",
        "? elevated-rails >= 2.0.8",
        "? quality >= 2.0.8"
    ],
```

# Base Game

<img width="474" alt="image" src="https://github.com/user-attachments/assets/d86a0f3c-3070-45e5-80b2-69659656b5da">


# Space Age

<img width="370" alt="image" src="https://github.com/user-attachments/assets/79670d4f-bd0a-43e1-8244-7d0a82950e4e">

**Comment by urda** [comment]
This should be fixed via 86b8ced865634ef52a77c1760214da1c8439813e
